packages/core/.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug Report
+about: Create a report to help us improve
+title: "fix: "
+labels: bug
+---
+
+**Description**
+
```

Comment:
nice 👍🏻

Reply:
should these files be located in the `<root>/package/<package>/.github` folder or in `<root>/.github`? I've never seen this approach, so I'm wondering.

Reply:
I'm assuming this was generated by `mason` or `verg_good_cli`.

Reply:
Yes this was generated by `mason`

Reply:
Moved `ISSUE_TEMPLATE` to root `.github` folder 04019550eef63754b559cda1a63ebe3550959028